pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -0,0 +1,133 @@
+import os
+import re
+import subprocess
+
+import click
+import koji
+import yaml
+from errata_tool import Erratum
+from pyartcd import constants
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+
+class OperatorSDKPipeline:
+    def __init__(self, runtime: Runtime, group: str, assembly: str, updatelatest: bool) -> None:
+        self.runtime = runtime
+        self._logger = runtime.logger
+        self.version = group.split("-")[1]
+        self.assembly = assembly
+        self.updatelatest = updatelatest
+        self.sdk = "operator-sdk"
+
+        self._jira_client = runtime.new_jira_client()
+        ocp_build_data_repo = runtime.new_github_client().get_repo("openshift/ocp-build-data")
+        release_file = ocp_build_data_repo.get_contents("releases.yml", ref=group)
+        release_yaml = yaml.load(release_file.decoded_content, Loader=yaml.FullLoader)
+        self.extra_ad_id, self.parent_jira_key = self.get_ad_jira_key(assembly, release_yaml)
```

Comment:
Those lines could be moved to `run`.

Reply:
The reason is that, things in the constructor is harder to make a unit test, and we will probably add try-except in `run` to capture the exception message and send it to slack or somewhere else.